gobblin-service/src/main/java/org/apache/gobblin/service/modules/flow/BaseFlowToJobSpecCompiler.java

```diff
@@ -219,8 +199,6 @@ private  AddSpecResponse onAddFlowSpec(FlowSpec flowSpec) {
   public AddSpecResponse onAddSpec(Spec addedSpec) {
     if (addedSpec instanceof FlowSpec) {
       return onAddFlowSpec((FlowSpec) addedSpec);
-    } else if (addedSpec instanceof TopologySpec) {
-      return onAddTopologySpec( (TopologySpec) addedSpec);
```

Comment:
is it wise to remove the ability to be a TS listener?

Reply:
being a TS listener feels of no use to me. There should be no TS listener in gaas.

Reply:
I equate topologies w/ executors in the flow graph.  since the FG can change dynamically w/o requiring a system restart, it doesn't seem out of the question to change the set of topos w/o a system restart either.

I agree that's not what we've done thus far, but it's arguably inconvenient to require two separate changes to define a new executor - one to the FG and one to the gaas configs.  if it were possible to do both together, I would personally find that appealing